utils/crates-io/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+//! crates-io-manager library
+
+mod manifest;
+mod publisher;
+pub mod rename;
+mod version;
+
+pub use self::{manifest::ManifestWithPath, publisher::Publisher, version::verify};
+use anyhow::Result;
+use std::process::{Command, ExitStatus};
+
+/// Packages need to be published.
+pub const PACKAGES: [&str; 20] = [
```

Comment:
split it into lists of
packages we need to publish: ...
helper crates we need to publish: ...